homeassistant/components/atag/config_flow.py

```diff
@@ -1,16 +1,17 @@
 """Config flow for the Atag component."""
-from pyatag import DEFAULT_PORT, AtagDataStore, AtagException
+from pyatag import DEFAULT_PORT, AtagException, AtagOne
 import voluptuous as vol
 
 from homeassistant import config_entries
-from homeassistant.const import CONF_DEVICE, CONF_HOST, CONF_PORT
+from homeassistant.const import CONF_DEVICE, CONF_EMAIL, CONF_HOST, CONF_PORT
 from homeassistant.core import callback
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 
 from . import DOMAIN  # pylint: disable=unused-import
 
 DATA_SCHEMA = {
     vol.Required(CONF_HOST): str,
+    vol.Optional(CONF_EMAIL): str,
```

Comment:
Where is this used?

Reply:
`atag = AtagDataStore(session, **user_input)`
and then downstream in the library, when making authenticated requests to the device.